cvFit/src/app/workoutList/workoutlist.component.html

```diff
@@ -9,6 +9,11 @@ <h1>
   </h1>
   <hr>
   <div class="flex-container" *ngIf='workouts$ | async as workouts'>
+    <div *ngIf="!workouts?.length">
+      <h1>No workouts to display!</h1>
+      <div class="center">
```

Comment:
Do not name your class names based on the style they produce. If some day you decide that this style should make them right aligned, you would feel like you also need to change the class name. Instead, call it based on what the element represents. In this case maybe it's `workout` or `create-workout-container`.